README.md

```diff
@@ -1,69 +1,108 @@
+[![Get help on Slack](http://img.shields.io/badge/slack-nf--core%20%23circrna-4A154B?labelColor=000000&logo=slack)](https://nfcore.slack.com/channels/circrna)[![Follow on Twitter](http://img.shields.io/badge/twitter-%40nf__core-1DA1F2?labelColor=000000&logo=twitter)](https://twitter.com/nf_core)[![Watch on YouTube](http://img.shields.io/badge/youtube-nf--core-FF0000?labelColor=000000&logo=youtube)](https://www.youtube.com/c/nf-core)
 
 ## Introduction
 
-The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It comes with docker containers making installation trivial and results highly reproducible.
+**nf-core/circrna** is a bioinformatics best-practice analysis pipeline for circRNA quantification, differential expression analysis and miRNA target prediction of RNA-Seq data.
+
+The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It uses Docker/Singularity containers making installation trivial and results highly reproducible. The [Nextflow DSL2](https://www.nextflow.io/docs/latest/dsl2.html) implementation of this pipeline uses one container per process which makes it much easier to maintain and update software dependencies. Where possible, these processes have been submitted to and installed from [nf-core/modules](https://github.com/nf-core/modules) in order to make them available to all nf-core pipelines, and to everyone within the Nextflow community!
+
+On release, automated continuous integration tests run the pipeline on a full-sized dataset on the AWS cloud infrastructure. This ensures that the pipeline runs on AWS, has sensible resource allocation defaults set to run on real-world datasets, and permits the persistent storage of results to benchmark between pipeline releases and other analysis sources.The results obtained from the full-sized test can be viewed on the [nf-core website](https://nf-co.re/circrna/results).
+
+## Pipeline summary
+
+1. Raw read QC ([`FastQC`](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/))
+2. Adapter trimming ([`Trim Galore!`](https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/))
+3. circRNA quantification
+   1. [`CIRIquant`](https://github.com/Kevinzjy/CIRIquant)
+   2. [`STAR 2-Pass mode`](https://github.com/alexdobin/STAR)
+      1. [`CIRCexplorer2`](https://circexplorer2.readthedocs.io/en/latest/)
+      2. [`circRNA finder`](https://github.com/orzechoj/circRNA_finder)
+      3. [`DCC`](https://github.com/dieterich-lab/DCC)
+   3. [`find circ`](https://github.com/marvin-jens/find_circ)
+   4. [`MapSplice`](http://www.netlab.uky.edu/p/bioinfo/MapSplice2)
+   5. [`Segemehl`](https://www.bioinf.uni-leipzig.de/Software/segemehl/)
+4. circRNA annotation
+5. Export mature spliced length as FASTA file
+6. Annotate parent gene, underlying transcripts.
+7. circRNA count matrix
```

Comment:
Are these the custom R scripts? Otherwise maybe also link the tools here in a similar fashion to above

Reply:
Nope, vanilla python and bash.